README.md

```diff
@@ -81,6 +81,44 @@ end
 
 Can be `http` or `https` protocol
 
+## Add your own service
+
+You can make available your own services:
```

Comment:
That sounds a bit weird. Usually, you say "you can make something available", not "you can make available something". I'd just go with "You can add/create your own services:".